lib/bolt/target.rb

```diff
@@ -286,15 +287,16 @@ def detail
       {
         'name' => name,
         'uri' => uri,
```

Comment:
I dont see a reason to print URI for `Target`. we do not expose that in inventory v1

Reply:
The name and URI can be different. For example if I have:
```
targets:
  - uri: dhix35pjb8hiya9.delivery.puppetlabs.net
    name: myhost
```

and run `bolt inventory show -n myhost --detail` I would want to know what the actual host it's using is. There's no way to find that without the URI.

Reply:
I only exclude URI for inventory v1 Target (v1). It is still included for v2.